src/components/SidePanel/BufferSettings.jsx

```diff
@@ -22,11 +22,11 @@ export const BufferSettings = () => {
     return (
         <CollapsibleGroup
             heading="Sampling Buffer"
-            title="Adjust max buffer size for sampling."
+            title="Adjust max buffer size for sampling"
         >
             <Form.Label
                 htmlFor="slider-ram-size"
-                title="Increase to sample for longer, decrease to solve performance issues."
+                title="Increase to sample for longer, decrease to solve performance issues"
```

Comment:
This should probably have been noted in the last PR, but it seems a bit inaccurate to say that decreasing will _solve performance issues_. I think we should instead write that this slider adjusts the amount of memory allocated.

Reply:
I think this should be discussed with Ketil